examples/astro-forms-demo/src/components/MultilineAddress.tsx

```diff
@@ -0,0 +1,168 @@
+import { type MultilineAddressProps } from '@wix/headless-forms/react';
+
+const MultilineAddress = ({
+  id,
```

Comment:
It is unclear how to implement this component, because the layout and fields are dynamic, based on country.